pilot/docker/envoy_pilot.yaml.tmpl

```diff
@@ -0,0 +1,286 @@
+        max_requests: 100000
+        max_retries: 3
+    connect_timeout: 1.000s
+    hosts:
+    - socket_address:
+        address: 127.0.0.1
+        port_value: 8080
+    name: in.8080
+  - circuit_breakers:
+      thresholds:
+      - max_connections: 100000
+        max_pending_requests: 100000
+        max_requests: 100000
+        max_retries: 3
+    connect_timeout: 1.000s
+    hosts:
+    - socket_address:
+        address: 127.0.0.1
+        port_value: 15010
+    http2_protocol_options: {}
+    name: in.15010
+  - circuit_breakers:
+      thresholds:
+      - max_connections: 100000
+        max_pending_requests: 100000
+        max_requests: 100000
+        max_retries: 3
+    connect_timeout: 1.000s
+    hosts:
+    - socket_address:
+        address: istio-telemetry
+        port_value: 15004
+    http2_protocol_options: {}
+    name: mixer_report_server
+{{- if .ControlPlaneAuth }}
+    tls_context:
+      common_tls_context:
+        tls_certificates:
+        - certificate_chain:
+            filename: /etc/certs/cert-chain.pem
+          private_key:
+            filename: /etc/certs/key.pem
+        validation_context:
+          trusted_ca:
+            filename: /etc/certs/root-cert.pem
+          verify_subject_alt_name:
+          - spiffe://cluster.local/ns/{{ .PodNamespace }}/sa/istio-mixer-service-account
+{{- end }}
+    type: STRICT_DNS
+  listeners:
+  - address:
+      socket_address:
+        address: 0.0.0.0
+        port_value: 15003
+    filter_chains:
+    - filters:
+      - config:
+          access_log:
+          - config:
+              path: /dev/stdout
+            name: envoy.file_access_log
+          http_filters:
+          - config:
+              default_destination_service: istio-pilot.{{ .PodNamespace }}.svc.cluster.local
+              service_configs:
+                istio-pilot.{{ .PodNamespace }}.svc.cluster.local:
+                  disable_check_calls: true
+                  mixer_attributes:
+                    attributes:
+                      destination.service:
+                        string_value: istio-pilot.{{ .PodNamespace }}.svc.cluster.local
+                      destination.uid:
+                        string_value: kubernetes://{{ .PodName }}.{{ .PodNamespace }}
+              transport:
+                check_cluster: mixer_check_server
+                report_cluster: mixer_report_server
+            name: mixer
+          - name: envoy.router
+          route_config:
+            name: "15003"
+            virtual_hosts:
+            - domains:
+              - '*'
+              name: istio-pilot.{{ .PodNamespace }}.svc.cluster.local
+              routes:
+              - decorator:
+                  operation: v1
+                match:
+                  prefix: /
+                route:
+                  cluster: in.8080
+                  timeout: 0.000s
+          stat_prefix: "15003"
+        name: envoy.http_connection_manager
```

Comment:
Q: are all names namespaced now?

Reply:
Yes, and they always start with envoy for default filters.